File: Java/BinarySearch/Single Among Doubles in a Sorted.java
//https://www.geeksforgeeks.org/problems/find-the-element-that-appears-once-in-sorted-array0624/1
class Solution {
    int findOnce(int[] arr) {
        // Complete this function
        int ans=0;

for(int i=0;i<arr.length;i++){

ans=ans^arr[i]; // using xor operation

}

return ans;
    }
}

File: Python/BinarySearch/Single Among Doubles in a Sorted.py
//https://www.geeksforgeeks.org/problems/find-the-element-that-appears-once-in-sorted-array0624/1
class Solution:
    def findOnce(self, arr):
        # Complete this function
        ans = 0
        for num in arr:
            ans ^= num  # Using XOR operation
        return ans
